journal-parsing/src/main/java/io/debezium/ibmi/db2/journal/retrieve/RetrieveJournal.java

```diff
@@ -111,9 +111,9 @@ public boolean retrieveJournal(JournalProcessedPosition previousPosition, final
         builder.init();
         builder.withBufferLenth(config.journalBufferSize());
         builder.withJournalEntryType(JournalEntryType.ALL);
-        if (config.filtering() && !config.includeFiles().isEmpty()) {
-            builder.withFileFilters(config.includeFiles());
-        }
+        // if (config.filtering() && !config.includeFiles().isEmpty()) {
+        // builder.withFileFilters(config.includeFiles());
```

Comment:
Oh we really do need this ability
We have clients with millions of events an hour and we absolutely need to filter.

Reply:
entries in journal that correspond to start commit and end commit has null object, and they are not processed by connector because of this filter
Need to think about how to make a filter that will not cause commit/rollback entries filtering